app/views/projects/confirm_delete.html.erb

```diff
@@ -0,0 +1,18 @@
+<header class="heading-with-actions">
+  <h1><%= project.name %></h1>
+</header>
+
+<div class="lead">
+  You are about to <strong>delete</strong> this project and the project content
+  items that belong to it - this will remove the project and any suggested tags
+  for content items from the system, and cannot be undone.
```

Comment:
It would be good to try and clarify this more, as I think "remove ... any suggested tags for content items from the system" sounds like it removes the tags that it has been used to create, which I don't think is the case.

Reply:
If Sona were here I'd check with her, but what do you think about the following:

> You are about to delete this project - this will remove the project and any participant suggested tags that may still be outstanding, and cannot be undone.